.eslintrc.js

```diff
@@ -25,19 +25,20 @@ module.exports = {
         'indent-legacy': 'error',
         'ember/named-functions-in-promises': 0,
         'function-paren-newline': ['error', 'consistent'],
+        'ember/no-attrs-snapshot': 0,
```

Comment:
The reason for it is that otherwise eslint complains about https://github.com/CenterForOpenScience/ember-osf-web/pull/165/commits/f19a8eefc00f552a7831366691d2ea4364e0443f#diff-81d9b662c026d799102fafc886209830R40

This rule is not clever enough to know that we are just trying to type `this`.

We should continue to avoid using the attrs snapshot, this rule just won't be able to enforce that.